### DISCUSSION TITLE
Valheim+ Issue 0.9.7

### DISCUSSION THREAD

**Discussion by madisontheboy** [lloesche/valheim-server-docker#302] [Q&A]
My server just updated to V+ 0.9.7 but it now does not load the mod any longer. It appears the V+ mod maker changed something because I had to reinstall V+ on my client as well. However, I can't figure out how to fix it on the server. Any help would be great!

**Comment by lloesche** [comment]
This should be auto-resolved by the container. From the logs it looks like the V+ makers messed up the release.

I just checked our servers logs and there I see:
```
Apr  5 19:15:13 /supervisord: valheim-updater INFO - Local ValheimPlus archive with update date 2021-03-23T10:33:01Z differs from date 2021-04-05T17:03:30Z or failed to successfully install - updating
Apr  5 19:15:13 /supervisord: valheim-updater DEBUG - [31698] - Downloading https://github.com/valheimPlus/ValheimPlus/releases/download/0.9.7/UnixServer.zip to /opt/valheim/dl/plus/UnixServer.zip
...
...
Apr  5 20:45:22 /supervisord: valheim-updater INFO - Local ValheimPlus archive with update date 2021-04-05T17:03:30Z differs from date 2021-04-05T18:37:53Z or failed to successfully install - updating
Apr  5 20:45:22 /supervisord: valheim-updater DEBUG - [2403] - Downloading https://github.com/valheimPlus/ValheimPlus/releases/download/0.9.7/UnixServer.zip to /opt/valheim/dl/plus/UnixServer.zip
```

So they released the same version twice but with different contents. Once around 19:14 CEST and then again 1.5h later.

If you read the logs further, in the original 0.9.7 archive the file `BepInEx/plugins/ValheimPlus.dll` was missing:
```
Apr  5 19:15:14 /supervisord: valheim-updater    creating: extracted/BepInEx/patchers/
Apr  5 19:15:14 /supervisord: valheim-updater    creating: extracted/BepInEx/plugins/
Apr  5 19:15:14 /supervisord: valheim-updater    creating: extracted/doorstop_libs/
```

but then 1.5h later it's there:
```
Apr  5 20:45:22 /supervisord: valheim-updater    creating: extracted/BepInEx/patchers/
Apr  5 20:45:22 /supervisord: valheim-updater    creating: extracted/BepInEx/plugins/
Apr  5 20:45:22 /supervisord: valheim-updater   inflating: extracted/BepInEx/plugins/ValheimPlus.dll
```

I'm assuming they realized their mistake shortly after pushing the 0.9.7 release and then instead of releasing a 0.9.7.1 hotfix decided to overwrite the already released version. 🤷‍♂️
